.github/workflows/python-package.yml

```diff
@@ -24,12 +24,15 @@ jobs:
       uses: actions/setup-python@v3
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install cydoomgeneric
       working-directory: ${{github.workspace}}/cydoomgeneric
```

Comment:
Either `working-directory: ${{github.workspace}}`, or this line might be completely redundant

Reply:
Indeed , there is no need to set the `working-directory` as the setup.py and pyproject.toml are at the top level